framework/auth/views.py

```diff
@@ -80,10 +80,10 @@ def forgot_password_post():
         if user_obj:
             #TODO: Remove this rate limiting and replace it with something that doesn't write to the User model
             now = datetime.datetime.utcnow()
-            last_attempt = user_obj.forgot_password_last_post or now - datetime.timedelta(seconds=FORGOT_PASSWORD_MINIMUM_TIME)
+            last_attempt = user_obj.forgot_password_last_post or now - datetime.timedelta(seconds=settings.FORGOT_PASSWORD_MINIMUM_TIME)
```

Comment:
Like this, for example. Saman has another PR which renames this variable to something more generic, and it's going to conflict as soon as it gets merged in. (I admit that I should have seen settings in there and done it right the first time; that's totally on me.)